src/main/java/edu/indiana/dlib/amppd/web/WorkflowResultFilterValues.java

```diff
@@ -8,13 +8,13 @@
 @Data
 public class WorkflowResultFilterValues {
 	private List<String> submitters;
-	private List<String> workflows;
-	private List<String> collections;
+	private List<Object> workflows;
+	private List<Object> collections;
 	private List<String> externalIds;
 	private List<String> items;
 	private List<String> files;
 	private List<String> steps;
-	private List<String> outputs;
+	private List<Object> outputs;
 	private List<GalaxyJobState> statuses;
```

Comment:
Why do we need to change String to Object here? I guess I didn't quite catch this part during our online review. Lets talk about it. Since I'm working on another jira that needs to add filters, I'd like to revert there changes so I can test on top of how the code used to behave. If you see any problems, pls let me know.